llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/VectorExt/IR/VectorExtAttrs.td

```diff
@@ -99,5 +99,202 @@ def LayoutAttr : IREEVectorExt_Attr<"Layout",
   }];
 }
 
-#endif // IREE_DIALECT_VECTOREXT_ATTRS
+def NestedAttr : IREEVectorExt_Attr<"Nested",
+      [ DeclareAttrInterfaceMethods<VectorLayoutInterface> ]> {
+  let mnemonic = "nested";
```

Comment:
Can we call it `nested_layout`? Nested on its own seems too non-specific to me. Don't want to spend forever on names though.